test/regexs.test.ts

```diff
@@ -140,6 +140,26 @@ const tests = {
         ],
     },
 
+    COMMA_SEPARATED_EMAILS_REGEX: {
+        valid: [
+            'test@example.com',
+            'test@example.com,foo@example.com',
+            'test@example.com, foo@example.com',
+            'test@example.com ,foo@example.com',
+            'test@example.com , foo@example.com,bar@example.com',
+            'test@example.com,a.b.c+123~@example.com,bar@example.com',
+        ],
+        invalid: [
+            'not-an-email,test@example.com',
```

Comment:
maybe let's add empty string? and some other extreme emails? newlines? tabs? e.g. text with no @ etc.